### PR TITLE
Rename Stepper builder methods

### LINKED ISSUE #684: replace Stepper::min/max with 'with_range'
I think these two methods are slightly too generally named, and I would like to be consistent with the new method on Slider.

### PR DESCRIPTION
**Pull request by Finnerale**
This fuses `Stepper::min` and `Stepper::max` to `Stepper::with_range` and renames `Stepper::step` as well as `Stepper::wrap` to `Stepper::with_step` and `Stepper::with_wrap` respectively.

Closes #684

### CODE REVIEW

**Comment by cmyr** [comment]
Looks good! `with_wrap` stands out as feeling a little weird, but 🤷‍♂

**Comment by cmyr** [comment]
To bikeshed: how do we feel about `with_wrapping` or `with_wraparound`?

**Comment by Finnerale** [comment]
`with_wraparound` seems good to me, should I change it to that or wait a little longer?

**Comment by xStrom** [comment]
You can just change it to `with_wraparound` now.

Not related to your work per se, but I was looking at this `Stepper` code and the wraparound doesn't even work. With wraparound set the `Stepper` can only have its value as the `min` or `max`.

Looking at the PR (#308) it seems that initially the code was maybe more functional.
```rust
if (*data - old_data).abs() > EPSILON {
    // callback
    (self.value_changed)(ctx, data, env);
} else if self.wrap {
```
However in a commit titled [Adress stepper review feedback](https://github.com/xi-editor/druid/pull/308/commits/4f76e71eb2001a33bd82980252498c7f5b950b6f) that was removed and changed to the current non-working code. Also that change doesn't seem to be based on any actual reviewer feedback, so I guess some sort of mistake.

Feel free to fix the wraparound code here, or if you'd rather not that's fine too and we'll just merge the method renames only.

**Comment by Finnerale** [comment]
For now I've just renamed `with_wrap` to `with_wraparound`

The functionality in itself is indeed completely broken, I think I will deal with that in a separate PR after I'm done with #632

File: druid/src/widget/stepper.rs [hunk 1]
```diff
@@ -57,26 +57,27 @@ impl Stepper {
...
+    /// The default step is 1.0
+    pub fn with_step(mut self, step: f64) -> Self {
         self.step = step;
         self
     }
 
     /// Set whether the stepper should wrap around the minimum/maximum values.
-    pub fn wrap(mut self, wrap: bool) -> Self {
+    ///
+    /// The default is no wrap
```
Comment (cmyr) [inline druid/src/widget/stepper.rs]:
```suggestion
    /// The default is `false`.
```

File: druid/src/widget/stepper.rs [hunk 2]
```diff
@@ -57,26 +57,27 @@ impl Stepper {
...
+    pub fn with_range(mut self, min: f64, max: f64) -> Self {
         self.min = min;
+        self.max = max;
         self
     }
 
     /// Set the steppers amount by which the value increases or decreases.
-    pub fn step(mut self, step: f64) -> Self {
+    ///
+    /// The default step is 1.0
```
Comment (cmyr) [inline druid/src/widget/stepper.rs]:
```suggestion
    /// The default step is `1.0`.
```

**Pull request merged**